src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -261,11 +261,10 @@ const formConfig = {
         ratedDisabilities: {
           title: 'Your Rated Disabilities',
           path: 'select-disabilities',
+          hideOnReview: true,
           uiSchema: {
             'ui:description': 'Please choose the disability that you’re filing a claim for increase because the condition has gotten worse.',
             disabilities: {
-              // Using StringField because it doesn't do much and we just need to render the widget.
```

Comment:
Issue created to track and discuss this work [here.](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10449)